cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/StatementHandler.kt

```diff
@@ -615,12 +620,40 @@ class StatementHandler(frontend: PythonLanguageFrontend) :
         return declStmt
     }
 
+    private fun addExternallyImportedToAnalysis(importName: Name) {
+        ctx?.let { ctx ->
+            var currentName = importName
+            while (currentName.isNotEmpty()) {
+                var importPath = currentName.toString().replace(language.namespaceDelimiter, "/")
+
+                language.fileExtensions.forEach { fileExtension ->
+
+                    // Includes a file in the analysis, if it has the root path and
+                    ctx.externalSources
+                        .firstOrNull {
+                            val relPath =
+                                it.relativeTo(
```

Comment:
you are basically relativizing the path twice, once inside getRootPath to get the root path and then again on the root path. You can just to the relativeOrNull check here and directly use the result if its non-null

Reply:
rewrote this using in two places to remove double raltivizing path and removed the now unnecessary utility function.